### PR TITLE
Improved Sequence Test Coverage

### PR DESCRIPTION
**Pull request by joswinter**
https://github.com/ProgrammingLife5/LifeTiles/issues/152

### CODE REVIEW

**Comment by albert-smit** [comment]
looks good

File: lifetiles-sequence/src/test/java/nl/tudelft/lifetiles/sequence/model/SequenceEntryTest.java [hunk 1]
```diff
@@ -62,4 +65,25 @@ public void setReferenceTest() {
         assertTrue(se1.isReference());
         se1.setReference(false);
     }
+
+    @Test
+    public void getMetadataTest() {
+        HashMap<String, String> metaData = new HashMap<String, String>();
```
Comment (jorenham) [inline lifetiles-sequence/src/test/java/nl/tudelft/lifetiles/sequence/model/SequenceEntryTest.java]: use `Map` as type

**Comment by jorenham** [comment]
Nice, +3.87% coverage

**Comment by alangerak** [comment]
looks good

**Comment by rutgervandenberg** [comment]
Looks good.

**Pull request merged**